src/inverseRobot.js

```diff
@@ -1,13 +1,43 @@
 'use strict';
 
-/*
+/**
+ * Situs inversus
+ * Transposition of internal organs is a variant of internal anatomy, when
+ * the organs are located in a mirror image.
+ * Something similar happened to our robot Kolli. His keys became values,
+ * and values became keys. Help to repair the robot. Create a 'inverseRobot'
+ * function that takes 'robot' as a parameter and returns a new object in which
+ * keys will change places with values.
+ *
+ * If any of the object values are repeated, return 'null'.
+ *
+ * Example:
+ *
+ * const kolli = { Kolli: 'name', 123: 'chipVer', 3: 'wheels' };
+ * const robert = { Robert: 'name', 123: 'chipVer', 113: 'chipVer' };
+ * inverseRobot(robert) === null
+ * inverseRobot(kolli) === { name: 'Kolli', chipVer: '123', wheels: '3' }
+ *
+ *
  * @param {object} robot
  *
  * @return {object}
-*/
-
+ */
 function inverseRobot(robot) {
-  // write code here
+  let duplicate = Object.values(robot);
+  const obj = {};
+
+  duplicate = duplicate.some((e, i, a) => a.indexOf(e) !== a.lastIndexOf(e));
```

Comment:
You are using the `duplicate` variable to check for duplicate values in the `robot` object. This is not correct, because the `duplicate` variable is an array that contains the values of the `robot` object. You should use the `hasOwnProperty` function to check if a property exists in an object.